instrumentation-api/src/test/groovy/io/opentelemetry/instrumentation/api/tracer/BaseTracerTest.groovy

```diff
@@ -21,6 +25,10 @@ class BaseTracerTest extends Specification {
   @Shared
   def root = Context.root()
 
+  @Shared
+  def ContextKey<Boolean> suppressContextKey = ContextKey.named("otel.suppress_instrumentation")
```

Comment:
is this used somewhere?

Reply:
nope, this is a draft PR, I'm not even sure I want to proceed with it, let me close it to not confuse anyone.